bfdd/bfdd_vty.c

```diff
@@ -282,7 +282,7 @@ static void _display_peer_json(struct vty *vty, struct bfd_session *bs)
 {
 	struct json_object *jo = __display_peer_json(bs);
 
-	vty_out(vty, "%s\n", json_object_to_json_string_ext(jo, 0));
+	vty_json(vty, jo);
 	json_object_free(jo);
```

Comment:
You need to remove all `json_object_free`, it is already called inside `vty_json`.

Reply:
Right, will fix.

Reply:
Fixed.